tooling/electron-mksnapshot/package.json

```diff
@@ -11,7 +11,8 @@
     "clean-deps": "rimraf node_modules",
     "clean": "rimraf dist",
     "test": "yarn test-unit",
-    "test-unit": "mocha -r @packages/ts/register --config ./test/.mocharc.js --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json",
+    "test-unit": "mocha test/unit/**/*.spec.ts -r @packages/ts/register --config ./test/.mocharc.js --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json",
+    "test-integration": "mocha test/integration/**/*.spec.ts -r @packages/ts/register --config ./test/.mocharc.js --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json",
```

Comment:
Can these mocha opts be deduplicated? Either by moving them to a command or `mocha.opts` file?

Reply:
Good call. Updated here: [`9ac5768` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/9ac57680068b11b55ee9c565d3df2f0c80892f0a)